Makefile

```diff
@@ -1,4 +1,5 @@
 PYTHON=$(shell which python)
+PYTHON_DEVELOP_ARGS=$(shell if ($(PYTHON) setup.py develop --help 2>/dev/null | grep -q '\-\-user'); then echo "--user"; else echo ""; fi)
```

Comment:
the `PYTHON_DEVELOP_ARGS` is not really related to this commit. Would you please extract it and add in a separate commit?